### ISSUE TITLE
Demo Issue: Improved Differentiation for A.1.1

### ISSUE THREAD

**Issue by david-boles** [duckietown/docs-duckiesky_high_school#34]
- We should put in another activity or possibly a cool video we have of the drone flying just in case the teacher can't show their drone in class for any reason
- Should we do the drone flight before or after safety lesson? (I personally like it on the first day if we think that's ok safety-wise)

**Comment by david-boles** [comment]
I think it should be fine safety-wise! It's just the teacher flying so they'll know how to do preflight checks and enforce that people wear safety glasses, etc...

**Comment by david-boles** [comment]
... further discussion ...

**Comment by GarrettWarren** [comment]
Agreed that a video alternative should be included here. Additionally, the lesson plan suggests flying outside; however, flying outdoors is not permitted with the drone since it does not have an RC transmitter that can be used to take control of the flight in case of emergency.
